### DISCUSSION TITLE
Formily 保存的场景部分表单项精准校验，其他表单项不填不校验，提交场景全量校验

### DISCUSSION THREAD

**Discussion by zmxue** [alibaba/formily#1558] [Q&A 2.x]
背景：表单上有保存、提交两个行动点，提交是全量的校验，保存时仅做部分的校验
详细描述：
demo:https://codesandbox.io/s/formily-2-with-prize-create-forked-qs3jo?file=/App.tsx
期望：点击保存的时候只校验奖品名称和需求方，其他字段不校验。具体场景：代理人填写，然后删除，触发了必填校验，但在保存的时候期望如果奖品名称和需求方合法，依旧可以提交

增强：其他表单项填了值校验合理性，不填不校验

**Comment by janryWang** [comment]
没懂你的需求，我看保存是局部校验了呀？你是希望不局部校验还是什么呢

**Reply by zmxue** [reply]
操作下代理人输入框，然后删除，触发了必填校验，再保存的时候期望如果奖品名称和需求方合法，依旧可以提交，这种场景目前是不行的

**Reply by janryWang** [reply]
必填校验是不会区分你是保存还是普通提交场景的，如果你希望保存绕过必填，你可以在保存的时候先clearErrors
